libs/core/src/hooks/hook.instance.ts

```
@@ -0,0 +1,26 @@
+
+  constructor(scope: ScopeEntry, providers: ProviderRegistryInterface, record: HookRecord, token: Token) {
+    super(scope, providers, record, token, record.metadata);
+  }
+
+  protected initialize(): Promise<void> {
+    return Promise.resolve(undefined);
+  }
+
+  async run(input: In, ctx: Ctx): Promise<void> {
+
+  }
```

ESLint

[error] Unexpected empty async method 'run'.

(@typescript-eslint/no-empty-function)